src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -844,7 +844,7 @@ type TaskSeq =
     static member takeWhileAsync: predicate: ('T -> #Task<bool>) -> source: TaskSeq<'T> -> TaskSeq<'T>
 
     /// <summary>
-    /// Returns a sequence that, when iterated, yields elements of the underlying sequence until the given
+    /// Returns a task sequence that, when iterated, yields elements of the underlying sequence until the given
```

Comment:
L854 predicate desc is wrong/confusing

Reply:
It says:

```
A function that evaluates to false when no more items should be returned.
```

about the predicate function for `takeWhile`. What is the confusing bit? But if we want to improve existing wording on existing xml doc, we should open a separate pull request instead.

Reply:
Also, we should then improve the `Seq.fsi` comments, as in, return the favor to the community and improve there as well, as that is where most of this comes from.

Reply:
Yeah there's definitely scope for that, but don't forget also.... they'll appreciate TaskEx work like AwaitTaskCorrect just as much (and thankfully there's a broader base of contributors, even if its nowhere near any dream level of course!)